src/components/content/heroes/aurora-hero.tsx

```diff
@@ -22,7 +23,9 @@ export const AuroraHero = ({
       <AuroraFullSize type={auroraType} />
 
       <TextContainer>
-        <HeroText title={title}>{children}</HeroText>
+        <HeroText hideMobileText={hideMobileText} title={title}>
```

Comment:
Discussion. No action required:

Is this flexible in usage or every hero behaves like that? If so I would prefer not to make it an option.
Or if it's the default case and there is only one scenario to keep it we could flip it and say "swapMobile=false" or similar.

Reply:
The text should be hidden only on the Career and About-Us page. So I think it is okay to only hide the text with the active flag.

Regarding your concerns with the search engines: I did some quick research and all I found out is that it's basically not a good idea to show different content on mobile and desktop. This can be interpreted as cloaking by search engines. For our specific case, where the content is completely identical, I have not found out anything. Do you have any experiences from other projects or an idea how to solve this better?